languages/csharp/reference/exercise-concepts/high-scores.md

```diff
@@ -0,0 +1,33 @@
+# Concepts of high-scores
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/high-scores/Example.cs)
+
+## General concepts
+### Object-oriented
+- classes: 
+    - the tested methods are defined in a class
+- methods: 
+    - methods arguments: the time is passed as an argument
+    - return values: returning a value from a method
+
+### General
+- scoping
+    - use `{` and `}` to denote scoping
+    - visibility: making tested method and tested class `public`
+    - visibility: making the list that contains the scores `private` and potentially `readonly`
```

Comment:
Isn't this a duplicate Concept? I'd prefer to only list each Concept once, so maybe you can combine the two lines?

Reply:
Yes, cleaned